openapi/p4pa-pu-bff.openapi.yaml

```
@@ -51,6 +51,29 @@ paths:
           description: Forbidden - Access denied
         '500':
           description: Internal Server Error
+  /operator/entities:
```

This API is inteded to return the list of organizations enabled of the current user
```suggestion
  /organizations:
```